Habinezajanvier/Capstone
# #5 adjust mobile view

#### What does this PR do?
Add cross device responsiveness
#### Description of Task to be completed?
- Adjust mobile view on landing page
- Adjust mobile view on resume page
- Adjust mobile view on login and sign pages
- Adjust mobile view of contact page 
#### How should this be manually tested?
- Clone the repo `git clone https://github.com/Habinezajanvier/my-brand-project.git`
- Switch the working directory `cd my-brand-project`
- Checkout the branch `git checkout ft-adjusting-responsiveness-#5`
- In `/UI/` folder open `index.html` using your preferred browser`
- Open developer tool available for your browser and toggle mobile view
> You should see page featue adjust to feat the available screen to maintain visibility
#### Any background context you want to provide?
- Starting `/UI/index.html` using liveServer should be helpful
- If **Chrome** is installed it may be preferred over others
#### What are the relevant pivotal tracker stories?
Click [here](https://trello.com/c/W9DgnSS9/5-adjusting-mobile-view) for relavant **Trello** story
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/44426710/88249628-4e16ab00-cca5-11ea-837f-6091e65c0649.png)
